default_group_hosts: ["mbot01","mbot02","mbot03","mbot04","mbot05","mbot06","mbot07","mbot08","mbot09","mbot10","mbot11","mbot12","mbot13","mbot14"]
